packages/edit-post/src/components/header/template-title/index.js

```diff
@@ -33,12 +41,33 @@ function TemplateTitle() {
 	}
 
 	return (
-		<span className="edit-post-template-title">
-			{
-				/* translators: 1: Template name. */
-				sprintf( __( 'Editing template: %s' ), templateTitle )
-			}
-		</span>
+		<Dropdown
+			position="bottom center"
+			className="edit-post-template-top-area"
+			contentClassName="edit-post-template-top-area__popover"
+			renderToggle={ ( { onToggle } ) => (
+				<>
+					<div className="edit-post-template-title">
+						{ __( 'About' ) }
+					</div>
+					<Button isSmall isTertiary onClick={ onToggle }>
+						{ templateTitle }
+					</Button>
+				</>
+			) }
+			renderContent={ () => (
+				<NavigableMenu>
+					<MenuGroup
+						className="edit-post-template-top-area__first-menu-group"
+						label={ __( 'Title' ) }
+						disabled
+					>
+						<MenuItem>{ templateTitle }</MenuItem>
+					</MenuGroup>
```

Comment:
This menu is clickable and focusable but doesn't actually do anything.
I suggest not adding this as a menu until there is an action.

Reply:
Agree, the user already clicked on a button with the template name, so making this focusable doesn't add anything.

Reply:
As mentioned [here](https://github.com/WordPress/gutenberg/pull/31678#issuecomment-842237213), this should be an input, not a menu item.